kong/plugins/rate-limiting/policies/init.lua

```diff
@@ -112,6 +135,72 @@ local function get_redis_connection(conf)
 end
 
 
+local function sync_to_redis(premature, conf)
+  if premature then
+    return
+  end
+
+  local red, err = get_redis_connection(conf)
+  if not red then
+    return nil, err
+  end
+
+  red:init_pipeline()
+
+  for cache_key, delta in pairs(cur_delta) do
+    red:eval([[
+      local key, value, expiration = KEYS[1], tonumber(ARGV[1]), ARGV[2]
```

Comment:
is `tonumber` necessary here or if we can just pass in a number?

Reply:
done